package.json

```diff
@@ -5,8 +5,9 @@
   "main": "src/index.ts",
   "scripts": {
     "test": "jest",
-    "dev": "node-dev src/index.ts",
+    "dev": "node-dev src/write-yaml-template.ts",
```

Comment:
to be removed in a later commit, I just didn't wanna have to run the wizard over and over to test this